packages/libs/react-components/src/components/Notification/styles.ts

```diff
@@ -0,0 +1,123 @@
+import { IconButton } from '../..';
+import { styled } from '../../styles';
+import { Button } from '../Button';
+import { Heading, Text } from '../Typography';
+
+export const NotificationColors: string[] = [
+  'default',
+  'primary',
+  'secondary',
+  'positive',
+  'warning',
+  'negative',
+];
```

Comment:
Is there a reason why you need this? These are already defined in the styled component so that can be used for typing